From 2532a43b676cc12de7ba4bd4d53e35090a46205f Mon Sep 17 00:00:00 2001
From: MartinEesmaa <martin.eesmaa@protonmail.com>
Date: Mon, 29 Apr 2024 22:48:30 +1000
Subject: [PATCH] Still support VVC1 fourcc from V_QUICKTIME

This will continue to detect to demux fourcc of vvc1 on V_QUICKTIME Codec ID of Matroska
---
 demux/demux_mkv.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/demux/demux_mkv.c b/demux/demux_mkv.c
index 9bee8a4011..2b3eba72aa 100644
--- a/demux/demux_mkv.c
+++ b/demux/demux_mkv.c
@@ -1539,6 +1539,13 @@ static int demux_mkv_open_video(demuxer_t *demuxer, mkv_track_t *track)
             extradata = track->private_data;
             extradata_size = track->private_size;
         }
+        if (fourcc1 == MKTAG('v', 'v', 'c', '1') ||
+            fourcc2 == MKTAG('v', 'v', 'c', '1'))
+        {
+            sh_v->codec = "vvc";
+            extradata = track->private_data;
+            extradata_size = track->private_size;
+        }
     } else {
         for (int i = 0; mkv_video_tags[i][0]; i++) {
             if (!strcmp(mkv_video_tags[i][0], track->codec_id)) {
-- 


